#!/workspaces/AN6001A-2024-T2-AI-BIGDATA/.conda/bin/python3.11
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
